.eslintrc.js

```diff
@@ -49,6 +50,10 @@ module.exports = {
     // allow debugger during development
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'brace-style': ["error", "stroustrup", { "allowSingleLine": true }],
+    // disable linebreak-style rule
+    'linebreak-style': 0,
```

Comment:
Please change "0" to 'off', as the syntax is clearer and more in line with eslint syntax convention.